roxctl/generate/netpol/cta.go

```diff
@@ -12,12 +17,82 @@ func (cmd *generateNetpolCommand) generateNetpol() error {
 		return errors.Wrap(err, "Error synthesizing policies from folder")
 	}
 
+	var yamlPolicies []string
```

Comment:
Could:

```suggestion
	yamlPolicies := make([]string, 0, len(recommendedNetpols))
```

Reply:
This is a unused bit of code that I removed, but that's a great suggestion, thanks!